lib/stove-testing-e2e-couchbase/src/main/kotlin/com/trendyol/stove/testing/e2e/couchbase/CouchbaseSystem.kt

```diff
@@ -95,6 +98,21 @@ class CouchbaseSystem internal constructor(
         .let(assertion)
         .let { this }
 
+    override suspend fun shouldNotGet(
```

Comment:
shouldNotGet naming looks not so clear for me.
Mb "ShouldNotExistById" or "ShouldNotExist" will be better

Reply:
I changed to ShouldNotExist